system-contracts/contracts/precompiles/EcAdd.yul

```diff
@@ -8,42 +14,32 @@ object "EcAdd" {
             //                      CONSTANTS
             ////////////////////////////////////////////////////////////////
 
-            /// @notice Constant function for value three in Montgomery form.
-            /// @dev This value was precomputed using Python.
-            /// @return m_three The value three in Montgomery form.
-            function MONTGOMERY_THREE() -> m_three {
-                m_three := 19052624634359457937016868847204597229365286637454337178037183604060995791063
-            }
-
-            /// @notice Constant function for the alt_bn128 field order.
-            /// @dev See https://eips.ethereum.org/EIPS/eip-196 for further details.
-            /// @return ret The alt_bn128 field order.
-            function P() -> ret {
-                ret := 21888242871839275222246405745257275088696311157297823662689037894645226208583
-            }
-
-            /// @notice Constant function for the pre-computation of R^2 % N for the Montgomery REDC algorithm.
-            /// @dev R^2 is the Montgomery residue of the value 2^512.
-            /// @dev See https://en.wikipedia.org/wiki/Montgomery_modular_multiplication#The_REDC_algorithm for further detals.
-            /// @dev This value was precomputed using Python.
-            /// @return ret The value R^2 modulus the curve field order.
-            function R2_MOD_P() -> ret {
-                ret := 3096616502983703923843567936837374451735540968419076528771170197431451843209
-            }
-
-            /// @notice Constant function for the pre-computation of N' for the Montgomery REDC algorithm.
-            /// @dev N' is a value such that NN' = -1 mod R, with N being the curve field order.
-            /// @dev See https://en.wikipedia.org/wiki/Montgomery_modular_multiplication#The_REDC_algorithm for further detals.
-            /// @dev This value was precomputed using Python.
-            /// @return ret The value N'.
-            function N_PRIME() -> ret {
-                ret := 111032442853175714102588374283752698368366046808579839647964533820976443843465
+            /// @dev The gas cost of processing ecadd circuit precompile.
+            function ECADD_GAS_COST() -> ret {
```

Comment:
We will need to make calculations to find correct constant here. It is not expected to be equal to 300 as on EVM